.changeset/afraid-ladybugs-smile.md

```diff
@@ -0,0 +1,7 @@
+---
+"@spectrum-css/thumbnail": major
```

Comment:
We'll have to run changeset again too, since we updated the opacity checkerboard css.

Reply:
Dropped this changeset and added a new one. Should we remove `thumbnail` from the new one if the new style is _just_ in the opacity checkerboard? ✨